aerospike-stubs/aerospike.pyi

```diff
@@ -442,6 +443,7 @@ class Query:
     def where(self, predicate: tuple, ctx: list = ...) -> None: ...
 
 class Scan:
+    ttl: int
```

Comment:
Do queries need this too?

Reply:
The type stubs already have `aerospike.Query.ttl`

https://github.com/aerospike/aerospike-client-python/blob/CLIENT-2645-client-conf-ttl/aerospike-stubs/aerospike.pyi#L431